packages/table/src/table/style-helper.ts

```diff
@@ -2,6 +2,7 @@ import { onMounted, onUnmounted, computed, ref, watchEffect, watch } from 'vue'
 import {
   addResizeListener,
   removeResizeListener,
+  ResizableElement,
```

Comment:
import type standalone, instead of in the actual import statement, like this:
```ts
import type { ResizableElement } from '@element-plus/utils/resize-event'
```

Reply:
OK